package.json

```diff
@@ -27,13 +27,10 @@
     "prepush:test": "jest --verbose --runInBand --onlyChanged",
     "test": "jest",
     "tsc": "tsc --noEmit",
-    "validate": "npm-run-all tsc lint test build"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm-run-all precommit:*",
-      "pre-push": "npm-run-all prepush:*"
-    }
+    "validate": "npm-run-all tsc lint test build",
+    "postinstall": "husky install",
+    "prepublishOnly": "pinst --disable",
+    "postpublish": "pinst --enable"
```

Comment:
I think these 2 scripts are failing our semantic releases. `pinst` is not found and thus giving error on pre-publish step

https://github.com/guardian/commercial-core/runs/2003924676?check_suite_focus=true

![Screenshot 2021-03-01 at 14 56 06](https://user-images.githubusercontent.com/51630004/109500067-68680100-7a9e-11eb-8936-af521d0b980e.png)